src/Components/Trending/Trending.jsx

```diff
@@ -0,0 +1,65 @@
+import "./Trending.scss";
+
+const Trending = () => {
+  return (
+    <>
+      <div className="head">
+        <h1>GUIDANCE GRID</h1>
+      </div>
+      <div className="searchbar">
+        <input type="text" placeholder="Search blogs" />
+      </div>
+      <secttion className="trending">
+        <h3>Trending Articles</h3>
+        <div className="trend-art1">
+          <div className="card">
+            <p className="art_name">
+              {" "}
+              01 <img src="./Images/profile.png" alt="" /> GDSC in Code.
```

Comment:
Pull the latest changes in the `home-page` branch and you will see a `public` directory. Place the images there creating a folder named `images`

`public/images`

[Checkout](https://vitejs.dev/guide/assets.html#the-public-directory) more about `public` directory in `vite`

Now you can access the assets placed in public directory as `/` because vite serves the public directory as root `/`

Suppose you have a file in the path

`public/images/profile.png`

It can be accessed in your HTML as 
```html
<img src="/images/profile.png">
```

Do the following changes
- Move the images from your component's directory to public directory.
- And access all the images as the way I mentioned above.